### PR TITLE
Make AWS Native example multi-language

### PR DESCRIPTION
**Pull request by infin8x**
- Fix gcp link
- Add all languages to AWS Native blog example

### CODE REVIEW

File: themes/default/content/blog/announcing-aws-native/index.md [hunk 1]
```diff
@@ -55,6 +59,110 @@ const objectlambda = new awsnative.s3objectlambda.AccessPoint("objectlambda-ap",
...
+{{% /choosable %}}
+
+{{% choosable language python %}}
+
+```python
+import pulumi
+import pulumi_aws_native as aws_native
+
+my_bucket = aws_native.s3.Bucket("myBucket")
+ap = aws_native.s3.AccessPoint("ap", bucket=my_bucket.id)
```
Comment (mikhailshilkov) [inline themes/default/content/blog/announcing-aws-native/index.md:71]: Maybe add empty line between resources?

File: themes/default/content/blog/announcing-aws-native/index.md [hunk 2]
```diff
@@ -33,6 +33,10 @@ The Pulumi AWS Native provider can be used in combination with the classic Pulum
 
 In this example, we can see how the new AWS S3 Object Lambda feature can be used via the AWS Native provider, with access to the full API defined by the S3 team at AWS, and combined naturally with another resource managed by the classic Pulumi AWS provider:
 
+{{< chooser language "typescript,python,csharp,go" >}}
+
+{{% choosable language typescript %}}
+
 ```typescript
 const bucket = new awsclassic.s3.Bucket("source");
```
Comment (lukehoban) [inline themes/default/content/blog/announcing-aws-native/index.md:41]: Should we update this to use awsnative as well given we're doing that in the other examples?

File: themes/default/content/blog/announcing-aws-native/index.md [hunk 3]
```diff
@@ -55,6 +59,110 @@ const objectlambda = new awsnative.s3objectlambda.AccessPoint("objectlambda-ap",
...
+        TransformationConfigurations =
+        {
+            new AwsNative.S3ObjectLambda.Inputs.AccessPointTransformationConfigurationArgs
+            {
+                Actions = { "GetObject" },
+                ContentTransformation = fn.Arn.Apply(arn => new Dictionary<string, object>
+                {
+                    ["AwsLambda"] = new Dictionary<string, object>
+                    {
+                        ["FunctionArn"] = fn.arn
```
Comment (mikhailshilkov) [inline themes/default/content/blog/announcing-aws-native/index.md:111]: It's a bit different in C#

```suggestion
                        ["FunctionArn"] = arn     
```

File: themes/default/content/blog/announcing-aws-native/index.md [hunk 4]
```diff
@@ -33,6 +33,10 @@ The Pulumi AWS Native provider can be used in combination with the classic Pulum
 
 In this example, we can see how the new AWS S3 Object Lambda feature can be used via the AWS Native provider, with access to the full API defined by the S3 team at AWS, and combined naturally with another resource managed by the classic Pulumi AWS provider:
```
Comment (lukehoban) [inline themes/default/content/blog/announcing-aws-native/index.md:34]:
```suggestion
In this example, we can see how the new AWS S3 Object Lambda feature can be used via the AWS Native provider, with access to the full API defined by the S3 team at AWS:
```

File: themes/default/content/blog/announcing-aws-native/index.md [hunk 5]
```diff
@@ -55,6 +59,110 @@ const objectlambda = new awsnative.s3objectlambda.AccessPoint("objectlambda-ap",
...
+import pulumi_aws_native as aws_native
+
+my_bucket = aws_native.s3.Bucket("myBucket")
+ap = aws_native.s3.AccessPoint("ap", bucket=my_bucket.id)
+objectlambdaap = aws_native.s3objectlambda.AccessPoint("objectlambdaap", object_lambda_configuration=aws_native.s3objectlambda.AccessPointObjectLambdaConfigurationArgs(
+    supporting_access_point=ap.arn,
+    transformation_configurations=[aws_native.s3objectlambda.AccessPointTransformationConfigurationArgs(
+        actions=["GetObject"],
+        content_transformation={
+            "awsLambda": {
```
Comment (mikhailshilkov) [inline themes/default/content/blog/announcing-aws-native/index.md:77]: I think you need to keep the casing
```suggestion
            "AwsLambda": {
```

File: themes/default/content/blog/announcing-aws-native/index.md [hunk 6]
```diff
@@ -55,6 +59,110 @@ const objectlambda = new awsnative.s3objectlambda.AccessPoint("objectlambda-ap",
...
+                "functionArn": fn.arn,
```
Comment (mikhailshilkov) [inline themes/default/content/blog/announcing-aws-native/index.md:78]:
```suggestion
                "FunctionArn": fn.arn,
```

File: themes/default/content/blog/announcing-aws-native/index.md (hunk 5 above)
Comment (lukehoban) [inline themes/default/content/blog/announcing-aws-native/index.md:77]: I believe this and the `functionArn` need to be PascalCase?

File: themes/default/content/blog/announcing-aws-native/index.md [hunk 7]
```diff
@@ -55,6 +59,110 @@ const objectlambda = new awsnative.s3objectlambda.AccessPoint("objectlambda-ap",
...
+        }
+        _, err = s3objectlambda.NewAccessPoint(ctx, "objectlambdaap", &s3objectlambda.AccessPointArgs{
+            ObjectLambdaConfiguration: &s3objectlambda.AccessPointObjectLambdaConfigurationArgs{
+                SupportingAccessPoint: ap.Arn,
+                TransformationConfigurations: []s3objectlambda.AccessPointTransformationConfigurationArgs{
+                    &s3objectlambda.AccessPointTransformationConfigurationArgs{
+                        Actions: pulumi.StringArray{
+                            pulumi.String("GetObject"),
+                        },
+                        ContentTransformation: pulumi.Any{
```
Comment (lukehoban) [inline themes/default/content/blog/announcing-aws-native/index.md:145]: I suspect ContentTransformation needs to be a pulumi.Map, and that the nested keys need to be PascalCase?

File: themes/default/content/blog/announcing-aws-native/index.md (hunk 2 above)
Comment (infin8x) [reply themes/default/content/blog/announcing-aws-native/index.md:41]:
```suggestion
const bucket = new awsnative.s3.Bucket("source");
```

File: themes/default/content/blog/announcing-aws-native/index.md [hunk 8]
```diff
@@ -55,6 +59,110 @@ const objectlambda = new awsnative.s3objectlambda.AccessPoint("objectlambda-ap",
...
+
+{{% choosable language python %}}
+
+```python
+import pulumi
+import pulumi_aws_native as aws_native
+
+my_bucket = aws_native.s3.Bucket("myBucket")
+ap = aws_native.s3.AccessPoint("ap", bucket=my_bucket.id)
+objectlambdaap = aws_native.s3objectlambda.AccessPoint("objectlambdaap", object_lambda_configuration=aws_native.s3objectlambda.AccessPointObjectLambdaConfigurationArgs(
```
Comment (infin8x) [inline themes/default/content/blog/announcing-aws-native/index.md:72]:
```suggestion
my_bucket = aws_native.s3.Bucket("myBucket")

ap = aws_native.s3.AccessPoint("ap", bucket=my_bucket.id)

objectlambdaap = aws_native.s3objectlambda.AccessPoint("objectlambdaap", object_lambda_configuration=aws_native.s3objectlambda.AccessPointObjectLambdaConfigurationArgs(
```

File: themes/default/content/blog/announcing-aws-native/index.md (hunk 5 above)
Comment (infin8x) [reply themes/default/content/blog/announcing-aws-native/index.md:77]: Sounds good. Note that `cf2pulumi` generated this code so either I mis-wrote my CloudFormation (very likely) or we may have a bug.

File: themes/default/content/blog/announcing-aws-native/index.md (hunk 7 above)
Comment (infin8x) [reply themes/default/content/blog/announcing-aws-native/index.md:145]: Would love help making this valid Go - this is beyond the scope of my current Go skills.

File: themes/default/content/blog/announcing-aws-native/index.md (hunk 5 above)
Comment (lukehoban) [reply themes/default/content/blog/announcing-aws-native/index.md]: I think there is likely a cf2pulumi bug in that is is overly-camelCasing these inputs.  Will confirm that in a bit and if so open an issue to track.

File: themes/default/content/blog/announcing-aws-native/index.md (hunk 7 above)
Comment (lukehoban) [reply themes/default/content/blog/announcing-aws-native/index.md:147]: I believe this works:

```go
		_, err = s3objectlambda.NewAccessPoint(ctx, "objectlambdaap", &s3objectlambda.AccessPointArgs{
			ObjectLambdaConfiguration: &s3objectlambda.AccessPointObjectLambdaConfigurationArgs{
				SupportingAccessPoint: ap.Arn,
				TransformationConfigurations: s3objectlambda.AccessPointTransformationConfigurationArray{
					&s3objectlambda.AccessPointTransformationConfigurationArgs{
						Actions: pulumi.StringArray{
							pulumi.String("GetObject"),
						},
						ContentTransformation: pulumi.Map{
							"AwsLambda": pulumi.Map{
								"FunctionArn": fn.Arn,
							},
						},
					},
				},
			},
		})
```

**Review by lukehoban — APPROVED** [review]

File: themes/default/content/blog/announcing-aws-native/index.md [hunk 9]
```diff
@@ -31,10 +31,14 @@ Resources available in the Pulumi AWS Native provider are based on the resources
...
 
-In this example, we can see how the new AWS S3 Object Lambda feature can be used via the AWS Native provider, with access to the full API defined by the S3 team at AWS, and combined naturally with another resource managed by the classic Pulumi AWS provider:
+In this example, we can see how the new AWS S3 Object Lambda feature can be used via the AWS Native provider, with access to the full API defined by the S3 team at AWS:
+
+{{< chooser language "typescript,python,csharp,go" >}}
+
+{{% choosable language typescript %}}
 
 ```typescript
-const bucket = new awsclassic.s3.Bucket("source");
```
Comment (joeduffy) [inline themes/default/content/blog/announcing-aws-native/index.md:37]: Maybe show the import statement here like the other languages?

**Pull request merged**